docs/quickstarts/openshift-ocp-2/openshift-ocp-library.yml

```diff
@@ -0,0 +1,16 @@
+apiVersion: console.openshift.io/v1
+kind: QuickStarts
+metadata:
+  name: openshift-ocp-library
```

Comment:
This name as to be `openshift-ocp-2`

It needs to match the name from the metadata file.

Reply:
ohhhhhhhhh ok! fixing